### DISCUSSION TITLE
How to get date only value from mysql date column?

### DISCUSSION THREAD

**Discussion by tsulatsitamim** [adonisjs/core#3298] [Help] [answered]
how to get ```"2021-10-08"``` instead of ```"2021-10-08T00:00:00.000Z"``` from mysql date column?
I use database query

**Comment by ruanitto** [comment]
You can use serialize method:

https://docs.adonisjs.com/guides/models/serializing-models#mutatingtransforming-values

```Javascript
@column.dateTime({
    autoCreate: true,
    serialize: (value: DateTime | null) => {
      return value ? value.toFormat('yyyy-MM-dd') : value
    },
  })
  public created_at: DateTime
```

**Comment by RomainLanz** [comment, marked as answer]
Hey @tsulatsitamim! 👋🏻 

When defining your column, use the `@column.date()` decorator.

```ts
import { DateTime } from 'luxon'
import { column, BaseModel } from '@ioc:Adonis/Lucid/Orm'

class User extends BaseModel {
  @column.date()
  public dob: DateTime
}
```

📚 https://docs.adonisjs.com/reference/orm/decorators#columndate--columndatetime

**Reply by tsulatsitamim** [reply]
I use Database query here not Model

**Reply by thetutlage** [reply]
> I have set column to regular string in model but still has time in response

Your original question mentioned you are using the Model. But anyways, you must share the proper code you have written to get proper help

**Reply by tsulatsitamim** [reply]
![image](https://user-images.githubusercontent.com/5946791/141883298-2fce90b4-0a24-4ffe-8625-91183b8f11bf.png)

![image](https://user-images.githubusercontent.com/5946791/141883348-583ff817-ef85-4f56-8548-bb2d8ef446cb.png)


I'm sorry i've edit my question. the screenshot above show my query
